core/src/main/java/bio/terra/pearl/core/service/kit/LivePepperDSMClient.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.pearl.core.service.kit;
+
+import bio.terra.pearl.core.model.kit.KitRequest;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.time.Instant;
+import java.util.Collection;
+import java.util.UUID;
+
+@Component
+public class LivePepperDSMClient implements PepperDSMClient {
+    private final PepperDSMConfig pepperDSMConfig;
+    private final WebClient webClient;
+    private final ObjectMapper objectMapper;
+
+    public LivePepperDSMClient(PepperDSMConfig pepperDSMConfig,
+                               WebClient.Builder webClientBuilder,
+                               ObjectMapper objectMapper) {
+        this.pepperDSMConfig = pepperDSMConfig;
+        this.webClient = webClientBuilder
+                .baseUrl(this.pepperDSMConfig.getBasePath())
+                .build();
+        this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public String sendKitRequest(Enrollee enrollee, KitRequest kitRequest, PepperKitAddress address)
+            throws JsonProcessingException {
+        var juniperKitRequest = PepperDSMKitRequest.JuniperKitRequest.builderWithAddress(address)
+                .juniperKitId(kitRequest.getId().toString())
+                .juniperParticipantId(enrollee.getShortcode())
+                .build();
+        var pepperDSMKitRequest = PepperDSMKitRequest.builder()
+                .juniperKitRequest(juniperKitRequest)
+                .kitType("SALIVA")
+                .juniperStudyId("Juniper-mock-guid")
```

Comment:
Still some details to work out here. These values should work based on info from Pegah, though I haven't gotten a kit request to work against DSM dev yet.